src/Roots/Acorn/Sage/Sage.php

```diff
@@ -43,7 +43,14 @@ class Sage
      *
      * @var ViewFactory
      */
-    protected $view;
+    protected          $view;
+
+    /**
+     * The Filesystem instance.
+     *
+     * @var Filesystem
+     */
+    private Filesystem $files;
```

Comment:
I try to avoid making things `private` unless there's compelling reason to do so.

```suggestion
    protected Filesystem $files;
```